src/test/java/dook/commands/Command.java

```diff
@@ -0,0 +1,19 @@
+package dook.commands;
+
+import java.io.IOException;
+
+import dook.DookException;
+import dook.storage.Storage;
+import dook.tasks.TaskList;
+import dook.ui.Ui;
+
+
+public abstract class Command {
```

Comment:
Good use of abstract class to extract out common attributes and methods for various command classes